packages/@sanity/core/src/actions/start/startAction.js

```diff
@@ -65,7 +65,8 @@ export default async (args, context) => {
 
     if (!hasErrors && !hasWarnings) {
       output.print(
-        chalk.green(`Compiled successfully! Server listening on http://${httpHost}:${httpPort}`)
+        chalk.green(`Sanity Content Studio compiled successfully!
+        Open it locally at http://${httpHost}:${httpPort}`)
```

Comment:
Is the line wrapping/indentation here intentional?

Reply:
The wrapping is (the linter complained), but I guess the indentation’s not.

Reply:
RIght, cause since this is targetted at a terminal, whitespace is actually respected. Probably better to split this up in a different way, or disable the max-length rule for the lines in question.